core/src/consensus/tendermint/worker.rs

```diff
@@ -1092,39 +1106,39 @@ impl Worker {
         SEAL_FIELDS
     }
 
-    fn generate_seal(&self, height: Height, parent_hash: BlockHash) -> Seal {
+    fn generate_seal(&mut self, height: Height, parent_hash: BlockHash) -> Seal {
         // Block is received from other nodes while creating a block
         if height < self.height {
             return Seal::None
         }
 
         // We don't know at which view the node starts generating a block.
         // If this node's signer is not proposer at the current view, return none.
```

Comment:
The above command does not match the current behavior. Let's remove it. Now every validator can be a proposer.

Reply:
Done